dotCMS/src/main/java/com/dotcms/graphql/exception/CustomGraphQLException.java

```diff
@@ -0,0 +1,39 @@
+package com.dotcms.graphql.exception;
```

Comment:
I think we could simply throw `GraphQLException` here and pass one or the other message instead

Reply:
yeah I informed myself a little bit better and now I can see why you did what you did here.